### PR TITLE
Update GraphStorm SageMaker solution to support Node ID remapping.

### PR DESCRIPTION
**Pull request by classicsong**
*Issue #, if available:*
#508 

*Description of changes:*
Update SageMaker support to support automatic node ID remapping.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: sagemaker/launch/common_parser.py [hunk 1]
```diff
@@ -5,6 +5,10 @@
 from ast import literal_eval
 import argparse
 
+# Npte: Sync with graphstorm.config SUPPORTED_TASKS
```
Comment (jalencato) [inline sagemaker/launch/common_parser.py:8]: Note：

Comment (jalencato) [reply sagemaker/launch/common_parser.py:8]: Btw, why not directly import from graphstorm.config

File: sagemaker/run/infer_entry.py [hunk 2]
```diff
@@ -50,6 +50,8 @@ def parse_train_args():
     parser.add_argument("--custom-script", type=str, default=None,
         help="Custom training script provided by a customer to run customer training logic. \
             Please provide the path of the script within the docker image")
+    parser.add_argument("--output-chunk-size", type=int, default=100000,
+        help="Number of rows per chunked prediction result or node embedding file.")
```
Comment (jalencato) [inline sagemaker/run/infer_entry.py:54]: Do we need to do something simlar for do-nid-remap？

File: sagemaker/launch/common_parser.py (hunk 1 above)
Comment (classicsong) [reply sagemaker/launch/common_parser.py:8]: The launch script will not rely on graphstorm package.

File: sagemaker/run/infer_entry.py (hunk 2 above)
Comment (classicsong) [reply sagemaker/run/infer_entry.py:54]: No. We have predefined behavior: Training will with --do-nid-remap False and Inference will with --do-nid-remap True.

**Review by jalencato — APPROVED** [review]
LGTM, will add to regression test later.

**Review by zheng-da — APPROVED** [review]

**Pull request merged**